backend/shelf-server/src/main/java/com/htec/shelfserver/controller/LoginController.java

```diff
@@ -1,20 +1,30 @@
 package com.htec.shelfserver.controller;
 
+import com.htec.shelfserver.annotation.AuthenticationUser;
+import com.htec.shelfserver.dto.AuthUser;
 import com.htec.shelfserver.dto.UserDTO;
 import com.htec.shelfserver.mapper.UserMapper;
 import com.htec.shelfserver.model.request.UserLoginMicrosoftRequestModel;
 import com.htec.shelfserver.model.request.UserLoginRequestModel;
+import com.htec.shelfserver.model.request.UserLogoutRequestModel;
+import com.htec.shelfserver.model.response.TextResponseMessage;
 import com.htec.shelfserver.model.response.UserLoginResponseModel;
 import com.htec.shelfserver.service.LoginService;
 import com.htec.shelfserver.util.TokenGenerator;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.web.authentication.logout.SecurityContextLogoutHandler;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
```

Comment:
Do you actually need all these imports ?

Reply:
Cleared unused imports